src/components/tags.js

```diff
@@ -1,4 +1,4 @@
-import styled from 'styled-components';
+import styled from 'react-emotion';
```

Comment:
Por que usar `react-emotion` en vez de `styled-components`?

Reply:
Es más rapido.

https://github.com/A-gambit/CSS-IN-JS-Benchmarks/blob/master/RESULT.md